src/playlist-maker.js

```diff
@@ -78,6 +78,38 @@ var playlistMaker = function(player, plist) {
     return currentIndex;
   };
 
+  // item can be either
+  //  * a string
+  //  * an array of sources, which are either strings or {src, type} objects
+  //  * a playlist item
+  playlist.contains = function contains(item) {
```

Comment:
Also, I feel like an `indexOf` should be slightly stricter than this `contains` since made it accept strings, array of sources and playlist items.